bluepyopt/ephys/locations.py

```diff
@@ -133,7 +133,7 @@ def instantiate(self, sim=None, icell=None):
     def __str__(self):
         """String representation"""
 
-        return '%s[%s]' % (self.seclist_name, self.sec_index)
+        return '%s: %s[%s]' % (self.name, self.seclist_name, self.sec_index)
```

Comment:
But that seems like a relevant piece of information.  One could also define two locations with different names, but the same values, and then tracking down which one emitted that string won't be possible.